core/src/schema/content/mod.rs

```diff
@@ -736,12 +736,5 @@ pub mod tests {
             "begin": "2020-11-05T09:53:10+0500",
             "end": "2020-11-05T09:53:10+0000"
         });
-
-        assert!(serde_json::from_value::<DateTimeContent>(json!({
```

Comment:
Datetime bounds are no longer enforced at the `serde` level (which makes total sense)

Reply:
Maybe we should take that direction with all the `serde` code and remove as much validation at the content tree level as we can, to leave it to be done in `compile` and `graph`?

Reply:
I think you're better off making this call and understanding the longer term repercussions of this decision. To me it does make sense - especially since we have some syntactic sugar happening at the serde level.